plugins/Actions/ArchivingHelper.php

```diff
@@ -385,28 +260,19 @@ private static function updateActionsTableRowWithGoals(array $row, bool $isPages
              return false;
         }
 
-        // Calculate adjusted revenue and conversions for page view goals
         if ($isPages &&
             isset($row[PiwikMetrics::INDEX_GOAL_NB_CONVERSIONS_ATTRIB]) &&
             isset($row[PiwikMetrics::INDEX_GOAL_NB_PAGES_UNIQ_BEFORE]))
         {
+            $row[PiwikMetrics::INDEX_GOAL_NB_CONVERSIONS_PAGE_RATE] = 0;
```

Comment:
Might be worth adding a comment, where that metric is calculated. Was just wondering why a metric should always be `0`. But it's simply calculated later in the `CalculateConversionPageRate` filter
```suggestion
            /**
             * Ensures this metric is available. It will be calculated later using a filter.
             * @see \Piwik\Plugins\Goals\DataTable\Filter\CalculateConversionPageRate
             */
            $row[PiwikMetrics::INDEX_GOAL_NB_CONVERSIONS_PAGE_RATE] = 0;
```